inc/networkport.class.php

```diff
@@ -264,7 +264,8 @@ function splitInputForElements($input) {
       $this->getEmpty();
 
       foreach ($input as $field => $value) {
-         if (array_key_exists($field, $this->fields)) {
+         if (array_key_exists($field, $this->fields)
+                 || $field == '_no_history') {
```

Comment:
Should be anough (better ?) to just skip all virtual fields

     $field[0] == '_'